### PR TITLE
release notes: 2.8.3

### PR DESCRIPTION
**Pull request by z7ye**

### CODE REVIEW

File: docs/source/release_notes.rst [hunk 1]
```diff
@@ -2,6 +2,13 @@
 Release Notes
 =============
 
+2.8.3
+-----
+Release date: March 21, 2023
+
+* Added support for BYOC and environment variables for :py:class:`~ads.model.GenericModel`.
+* Added default values for configuring parameters in :py:class:`~ads.model.ModelDeployment`, such as default flex shape, ocpus, memory in gbs, bandwidth, instance count.
```
Comment (jdesanto) [inline docs/source/release_notes.rst:10]:
```suggestion
* Added default values for configuring parameters in :py:class:`~ads.model.ModelDeployment`, such as default flex shape, ocpus, memory in gbs, bandwidth,  and instance count.
```

**Review by jdesanto — DISMISSED** [review]
One small addition.

File: docs/source/release_notes.rst [hunk 2]
```diff
@@ -2,6 +2,13 @@
...
+
```
Comment (qiuosier) [inline docs/source/release_notes.rst:11]:
```suggestion
* Added support for :py:class:`~ads.jobs.NotebookRuntime` to use directory as job artifact.
* Added support for :py:class:`~ads.jobs.PythonRuntime` and :py:class:`~ads.jobs.GitPythonRuntime` to use shell script as entrypoint.

```

Comment (z7ye) [reply docs/source/release_notes.rst:11]: done

File: docs/source/release_notes.rst (hunk 1 above)
Comment (z7ye) [reply docs/source/release_notes.rst:10]: fixed.

File: docs/source/release_notes.rst [hunk 3]
```diff
@@ -2,6 +2,13 @@
 Release Notes
 =============
 
+2.8.3
+-----
+Release date: March 21, 2023
+
+* Added support for BYOC and environment variables for :py:class:`~ads.model.GenericModel`.
```
[the remark this answers is not in the document]
Comment (z7ye) [reply docs/source/release_notes.rst:9]: fixed.

**Review by qiuosier — APPROVED** [review]

**Review by lvallad — APPROVED** [review]
looks good.

**Pull request merged**